app/src/main/java/com/duckduckgo/app/statistics/api/PixelService.kt

```diff
@@ -32,6 +32,7 @@ interface PixelService {
         @Path("formFactor") formFactor: String,
         @Query(AppUrl.ParamKey.ATB) atb: String,
         @QueryMap additionalQueryParams: Map<String, String?> = emptyMap(),
+        @QueryMap(encoded = true) encodedQueryParams: Map<String, String?> = emptyMap(),
```

Comment:
We need to not encode some values when we are sending them as a parameter hence the need to introduce this.
See https://app.asana.com/0/1125189844152671/1165694058809791 for more information.